apps/charterafrica/src/components/FeaturedPostCard/FeaturedPostCard.js

```diff
@@ -93,6 +93,7 @@ const FeaturedPostCard = React.forwardRef(function FeaturedPostCard(
             <RichText
               sx={{
                 maxHeight: 24 * 6,
+                overflow: "hidden",
```

Comment:
`lineClamp` should handle this. What's the issue?

Reply:
I am facing this.

![image](https://user-images.githubusercontent.com/39160236/222711303-8945eb46-f449-4875-af6a-a3dbd03fc86c.png)

payload 
```js
[
{
"children": [
{
"text": "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
"children": null
}
]
},
{
"children": [
{
"children": null,
"text": "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."
}
]
},
{
"children": [
{
"children": null,
"text": ""
}
]
},
{
"children": [
{
"children": null,
"text": "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."
}
]
},
{
"children": [
{
"children": null,
"text": ""
}
]
},
{
"children": [
{
"children": null,
"text": ""
}
]
},
{
"children": [
{
"children": null,
"text": ""
}
]
}
]
```

Reply:
Couple of things @koechkevin :

1. I believe lineClamp happens where there is ....
2. Fact that here is another line starting after that, it tells me it's probably another paragraph or something.

Excerpt should just be one paragraph so may be we should be passing that?